### DISCUSSION TITLE
feat: searching event based on titles and descriptions

### DISCUSSION THREAD

**Discussion by love98ooo** [NJUPT-SAST/sast-evento-backend#84] [Q&A]
在数据库中把搜索的内容找出来的难度不大，但找出返回可能的相关结果才是需要解决的难点，此外还有返回的排序问题

**Comment by feellmoose** [comment]
Full-Text Search 确实是解决中等数据量文本进行搜索的很好的选择，支持搜索可能相关的结果，并支持增加或降低相关性。
